Kitodo/src/main/webapp/WEB-INF/web.xml

```diff
@@ -124,6 +124,10 @@
     <filter>
         <filter-name>requestControlFilter</filter-name>
         <filter-class>de.sub.goobi.helper.servletfilter.RequestControlFilter</filter-class>
+        <init-param>
+            <param-name>excludePattern</param-name>
+            <param-value>.*\.(js|css)\.jsf</param-value>
```

Comment:
> This pattern looks a bit strange. There should no file name ending of .js.jsf or .css.jsf if I understood this pattern correct.
> 
> I think you would only exclude .js and .css files. Then should the added \.jsf to be removed.

It looks strange but is the correct pattern to exclude for the `requestControlFilter`. Javascript and CSS resource files that come bundled with component libraries like PrimeFaces result in resource names like  `primefaces.js.jsf` in Facelets that use tags from the corresponding libraries.

Requests to these 'resources' should definitely not be processed by the `requestControlFilter`, because it often prevents their loading after a few calls to the same page, disabling any Javascript functionality and rendering the whole page unusable.

Individual Javascript files that don't come as part of a JSF component or tag library are not affected by this (for example the `toggle.js` file in `head.xhtml` included via a normal `<script>` tag), can therefore safely be processed by the filter and don't lead to the resource error described above.